aws/sdk/examples/apigateway/src/bin/get_rest_apis.rs

```diff
@@ -18,6 +18,28 @@ struct Opt {
     verbose: bool,
 }
 
+// Displays the Amazon API Gateway REST APIs in the Region.
+async fn show_apis(client: &aws_sdk_apigateway::Client) -> Result<(), aws_sdk_apigateway::Error> {
```

Comment:
The client and error are already imported:
```suggestion
async fn show_apis(client: &Client) -> Result<(), Error> {
```

Reply:
Good catch, fixed.